pyasic/config/temperature.py

```diff
@@ -39,6 +39,17 @@ def as_bosminer(self) -> dict:
             temp_cfg["dangerous_temp"] = self.danger
         return {"temp_control": temp_cfg}
 
+    def as_epic(self) -> dict:
+        temp_cfg = {}
+        if self.target is not None:
+            temp_cfg["target_temp"] = self.target
+        else:
+            ## If no target_temp set, autoset to -15 of shutdown
+            temp_cfg["target_temp"] = self.hot - 15.0
```

Comment:
Good point, need to see what the FW will do. So desire is to send a null value to the FW and it rejects it? This one gets a bit complicated as its tied to autofan, so the other option is to set an autofan temp in the fan class and avoid using this for that purpose

Reply:
A good example is BOS+, in their config if you set only target temperature, they will set hot and danger for you depending on the model, but they have defaults for all these values.  That's my preferred way of doing it, since if you dont know what you want to set here the firmware picks a sensible default based on the machine.